utils/errorreport_test.go

```diff
@@ -0,0 +1,86 @@
+package utils
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli-security/tests"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"os"
+	"testing"
+)
+
+const (
+	unsupportedXscVersionForErrorLogs = "1.6.0"
+	supportedXscVersionForErrorLogs   = "1.7.1"
+)
+
+func TestReportToCoralogix(t *testing.T) {
+	serverDetails := &config.ServerDetails{
+		Url:            *tests.JfrogUrl,
+		ArtifactoryUrl: *tests.JfrogUrl + tests.ArtifactoryEndpoint,
+		XrayUrl:        *tests.JfrogUrl + tests.XrayEndpoint,
+		AccessToken:    *tests.JfrogAccessToken,
+		ServerId:       tests.ServerId,
+	}
+
+	// Prior to initiating the test, we verify whether Xsc is enabled for the customer. If not, the test is skipped.
+	xscManager, err := CreateXscServiceManager(serverDetails)
+	assert.NoError(t, err)
+
+	if !reportLogErrorEventPossible(xscManager) {
+		t.Skip("Skipping test since Xsc server is not enabled or below minimal required version")
+	}
+
+	errorToReport := errors.New("THIS IS NOT A REAL ERROR! This Error is posted as part of TestReportToCoralogix test")
+	assert.NoError(t, ReportToCoralogix(serverDetails, errorToReport, "cli"))
+}
```

Comment:
this is an integration test, move it to `xsc_test.go` at root

Reply:
?